Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -315,6 +312,7 @@ void ABenchmarkGymGameModeBase::TickPlayersConnectedCheck(float DeltaSeconds)
 
 	if (RequiredPlayerCheckTimer.HasTimerGoneOff() && !DeploymentValidTimer.HasTimerGoneOff())
 	{
+		int SmoothedTotalAuthPlayers = TotalActorCounts[SimulatedPawnClass].GetSmoothedTotal();
```

Comment:
We could consider getting rid of the TickPlayersConnectedCheck all together. Currently it simply checks the actor count of the player BP. Admittedly it is using the smoothed value rather than absolute... but I am sure we could come up with a way of configuring this in the actor count metric.